src/components/application_manager/test/rpc_service_impl_test.cc

```diff
@@ -78,6 +79,7 @@ const std::string kPolicyAppId = "policy_app_id";
 const uint32_t kCorrelationId = 1u;
 const uint32_t kFunctionId = 1u;
 const uint32_t kAppId = 1u;
+const uint32_t kProtocolType = 1u;
```

Comment:
I created 2 new variables because of problem with access to class with original definition of this constants. Fixed in commit fe444ecfee751ff4800cf80c77d53347f383288f